cvat-ui/webpack.config.js

```diff
@@ -50,13 +50,13 @@ module.exports = {
                 },
             },
         }, {
-            test: /node_modules\/antd\/[\w\/]*.less$/,
-            use: ['style-loader', 'css-loader', {
-                loader: 'less-loader',
+            test: /node_modules\/antd\/[\w\/]*.css$/,
+            use: ['style-loader', {
+                loader: 'css-loader',
                 options: {
-                    javascriptEnabled: true,
+                    importLoaders: 2,
                 },
-            }]
+            }, 'postcss-loader']
```

Comment:
Maybe just removing ``exclued: /node_modules/`` in the rule below will work?

Or another option is to change ``importLoaders`` to ``1`` as it specified [here](https://github.com/webpack-contrib/css-loader#importloaders)